test/integration/indexer/links_lookup_test.rb

```diff
@@ -80,6 +80,7 @@ def test_indexes_documents_with_links_from_publishing_api
       "specialist_sectors" => ["my-topic/a", "my-topic/b"],
       "mainstream_browse_pages" => ["my-browse/1"],
       "organisations" => ["my-org/1", "my-court"],
+      "part_of_taxonomy" => ["TAXON-1"],
```

Comment:
It would be nice to add a full test of the behaviour here. So given a taxon with parents a & b, the  `part_of_taxonomy` of taxon c will be a, b & c.